tests/TestCase/Model/SearchTraitTest.php

```diff
@@ -63,8 +65,8 @@ public function testFinder()
 
         $query = $this->Articles->find('search', ['search' => $queryString]);
         $this->assertSame([
-            'Articles.foo' => 'a',
-            'Articles.public' => false,
+            'foo' => 'a',
```

Comment:
This proves somewhat that there can be some BC break here in these changes (as so far it properly aliased).

Reply:
It's not BC break. The test was just wrong. `$this->Articles` is an instance of `Endpoint` class of `Webservice` plugin which doesn't have any `aliasField()` method.

Reply:
Interesting. OK

Reply:
Hmm no, I am wrong. `Endpoint` does have `aliasField()` method. So need to revert this :disappointed:

Reply:
Yeah that was my gut feeling / expectation here.